README.md

```diff
@@ -79,16 +79,16 @@ metadata:
   name: hmc
 spec:
   core:
-    capi:
-      template: cluster-api
+    capi: {}
```

Comment:
I think we can omit both capi and hmc if we don't override anything in this example